image/image.css

```diff
@@ -0,0 +1,35 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap");
+
+* {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
Suggestion reusable variables and colors can be kept in one file. eg: variables.css.

Reply:
Same suggestion as Khyati. Better to use reusable colors as variables and make it a separate css file